### ISSUE TITLE
Conversion from packed_arrayN type to Bigarray's ArrayN.t type

### ISSUE THREAD

**Issue by Octachron** [LaurentMazare/npy-ocaml#2]
Looking at npy-ocaml more or less by accident, I was thinking that it may be nice to add a function

```OCaml
val to_bigarrayN: 'c Bigarray.layout -> ('a,'b) kind -> package_arrayN -> ('a,'b,'c) Bigarray.ArrayN.t option
```
(for N∈{1,2,3} and Genarray) that will return `Some a` if the type of the packed array matches the types of the layout and kind argument and return `None` otherwise. Such function may ease the manipulation of the underlying bigarray. Would you be interested in an implementation of such functions?

**Comment by LaurentMazare** [comment]
Sure, having such functions could be convenient. Feel free to submit a PR for this (or if you prefer I can have a look myself).

**Comment by gasche** [comment]
Can this be closed now that #3 was merged?
